### PR TITLE
Chore/sentry updates

### PR DESCRIPTION
**Pull request by ajacksified**
* Updates sentry
* Adds performance traces to sentry
* Fixes error with the error pages and the error alert, in a fit of irony

### CODE REVIEW

File: src/contexts/alertContext/index.jsx [hunk 1]
```diff
@@ -39,14 +39,28 @@ function AlertProvider({ children }) {
   const [alertOpen, setAlertOpen] = useState(false);
   const [message, setMessage] = useState("");
   const [severity, setSeverity] = useState("success");
+
   const alertSuccess = (msg) => {
+    let errorMessage = msg;
+
+    if (msg instanceof Error) {
+      errorMessage = msg.toString();
```
Comment (megoth) [inline src/contexts/alertContext/index.jsx:47]: Maybe write a test for this branch?

File: src/contexts/alertContext/index.jsx [hunk 2]
```diff
@@ -39,14 +39,28 @@ function AlertProvider({ children }) {
...
-    setMessage(msg);
+    setMessage(errorMessage);
     setAlertOpen(true);
   };
+
   const alertError = (msg) => {
+    let errorMessage = msg;
+
+    if (msg instanceof Error) {
+      errorMessage = msg.toString();
```
Comment (megoth) [inline src/contexts/alertContext/index.jsx:59]: Ditto

**Review by megoth — CHANGES_REQUESTED** [review]
Would like a couple of tests, otherwise looks good.

**Comment by ajacksified** [comment]
I promise to add tests this afternoon - this fix is fairly urgent while NSS is throwing `500` errors.

**Review by megoth — APPROVED** [review]

**Pull request merged**